### PR TITLE
Create iCloud Error screen.

### PR DESCRIPTION
**Pull request by RomanEsin**

### CODE REVIEW

**Comment by RomanEsin** [comment]
Should there be a screen for granting iCloud access in onboarding view?

File: Project SF/Views/Onboarding/GrantDataAccessView.swift [hunk 1]
```diff
@@ -39,15 +39,15 @@ struct GrantDataAccessView: View {
...
+                .padding()
             Text("This application needs access to your health data to calculate the points in competitions.")
                 .padding(.horizontal, 16)
                 .font(.caption)
                 .foregroundColor(.secondary)
                 .multilineTextAlignment(.center)
         }
         .padding(.horizontal)
-        .navigationTitle("Health Data")
+        .navigationTitle("iCloud")
```
Comment (Priva28) [inline Project SF/Views/Onboarding/GrantDataAccessView.swift]: Why change nav title to iCloud if you aren't doing iCloud on this page?

**Review by Priva28 — APPROVED** [review]

File: Project SF/Views/Onboarding/GrantDataAccessView.swift (hunk 1 above)
Comment (RomanEsin) [reply Project SF/Views/Onboarding/GrantDataAccessView.swift]: Whoops, forgot to change that. I was just trying to add iCloud grant data on the onboarding.

Comment (RomanEsin) [reply Project SF/Views/Onboarding/GrantDataAccessView.swift]: Fixed

**Pull request merged**